cmd/pipeline/main.go

```diff
@@ -544,9 +544,8 @@ func main() {
 	router.Use(ocmux.Middleware())
 
 	// These two paths can contain sensitive information, so it is advised not to log them out.
-	skipPaths := config.Audit.SkipPaths
 	engine.Use(correlationid.Middleware())
-	engine.Use(ginlog.Middleware(logrusLogger, skipPaths...))
+	engine.Use(ginlog.Middleware(logrusLogger, []string{"/auth/dex/callback", "/pipeline/api"}...))
```

Comment:
The skip path configuration has been abused before this change: although it belonged to the audit log, we used it for filtering the access log as well. They are not the same use case though, in fact, I don't think we need to skip any paths in the access log middleware....it's access log after all. It doesn't log the request body. Also, they filter different paths now.

But changing that is out of scope for this PR, so for now I replaced it with the default value of the configuration. (Feel free to address that in a separate issue/PR though)

The reason why you see two places with that change is that we have an "internal API" that's not really used anymore (technically hollowtrees relies on it, but it's essentially dead), it was badly constructed in the first place, which is why it instantiates similar middleware. Addressing this is also out of scope for this PR in my opinion. (And I don't really care if it diverges from the original one, since it doesn't even serve those paths, so it's essentially non-functional)

I also think that exposing the skipped paths as a configuration was a mistake. I don't see any use case for the *user* to specify which paths should not have the request body recorded. It's rather internal to the application.